expression/builtin_arithmetic_vec.go

```diff
@@ -243,11 +244,116 @@ func (b *builtinArithmeticIntDivideIntSig) vecEvalInt(input *chunk.Chunk, result
 }
 
 func (b *builtinArithmeticPlusIntSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinArithmeticPlusIntSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	lh, err := b.bufAllocator.get(types.ETInt, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(lh)
+
+	if err := b.args[0].VecEvalInt(b.ctx, input, lh); err != nil {
+		return err
+	}
+
+	// reuse result as rh to avoid buf allocate
+	result.ResizeInt64(n, false)
```

Comment:
We need not call `ResizeInt64` here, the implementation of `b.args[1].VecEvalInt` has the responsibility to allocate memory for `result`.

Reply:
Thanks for your suggestion.